src/plugins/calibration/calibration_impl.cpp

```diff
@@ -220,8 +220,9 @@ void CalibrationImpl::cancel() const
     // All params 0 signal cancellation of a calibration.
     MavlinkCommandSender::CommandLong::set_as_reserved(command.params, 0.0f);
     command.target_component_id = target_component_id;
-    // We don't care about the result, the initial callback should get notified about it.
     _parent->send_command_async(command, nullptr);
+
+    return Calibration::Result::Success;
```

Comment:
@julianoes: I added this so that it builds, but I don't really get what I should be doing there, following your change in the proto file. Should I block until a result is received and return it? :thinking:

Reply:
I suggest a future promise around it, and if sending fails then you return `ConnectionError`.